CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+<!---
+## 0.0.1 - 1970-01-01
+
+### Added
+
+- New stuff.
+
+### Changed
+
+- Changed stuff.
+
+### Deprecated
+
+- Deprecated stuff.
+
+### Removed
+
+- Removed stuff.
+
+### Fixed
+
+- Fixed stuff.
+
+### Security
+
+- Security related fix.
+-->
+
+## Unreleased
+
+## 1.0.0 - 2019-10-12
```

Comment:
A version 1.0.0 was released ? I can only find the 0.0.X-dev0 on the test pypi server

Reply:
Oops, this is a mistake when I tried the `bump_version.sh` script I guess.